packages/web-components/src/components/va-icon/va-icon.scss

```diff
@@ -0,0 +1 @@
+@use 'usa-icon/src/styles/_usa-icon.scss';
```

Comment:
I think that we also want to add `@forward 'settings';` to the top of this file so that our predefined USWDS settings are included.

Reply:
done!